evcouplings/compare/sifts.py

```diff
@@ -106,14 +109,14 @@ def fetch_uniprot_mapping(ids, from_="ACC", to="ACC", format="fasta"):
     return r.text
 
 
-def find_homologs_jackhmmer(**kwargs):
+def find_homologs(use_jackhmmer=False, **kwargs):
```

Comment:
use_jackhmmer shouldn't be an explicit parameter here but also be accessed through kwargs; it is a config parameter like all the others